src/components/SkipTo.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <button class="skip-button" @click="skipToTarget"><slot></slot></button>
```

Comment:
depending how often we're going to need this, maybe we can implement this in the design system ?

Reply:
If the interface stays like this (the navigation being hidden away off-screen initially anyway) only one instance of a skip link (the one going to main) is necessary, in my opionion. Regarding other thoughts on SkipTo and ODS, see "Why no component in ODS?" above.